app/uk/gov/hmrc/gform/views/main_template.scala.html

```diff
@@ -34,7 +34,8 @@
   frontendAppConfig: FrontendAppConfig,
   authConfig: Option[AuthConfig],
   webChat: Option[WebChat] = None,
-  formCategory: FormCategory = Default
+  formCategory: FormCategory = Default,
+  templateId: String
```

Comment:
Let's keep type information here ie. use `FormTemplateId` instead of `String`

Reply:
Done